e2e2/test/cases/nvidia/manifests/mpi-job-nccl-test-multi-node.yaml

```diff
@@ -69,37 +69,33 @@ spec:
             - 2G
             - -f
             - "2"
-            - -t
-            - "1"
-            - -g
-            - "1"
```

Comment:
What did these do?

Reply:
The default values are 1.
https://github.com/NVIDIA/nccl-tests